ShareBook/ShareBook.Domain/Enums/EProfile.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ShareBook.Domain.Enums
+{
+    public enum EProfile
```

Comment:
Baseado nas convenções de nomenclatura, não deveria existir esse E antes do Profile.

https://msdn.microsoft.com/en-us/library/4x252001(v=vs.71).aspx
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces#naming-enumerations